expo-maps/android/src/main/java/expo/modules/maps/GoogleMapsGestures.kt

```diff
@@ -0,0 +1,26 @@
+package expo.modules.maps
+
+import com.google.android.gms.maps.GoogleMap
+
+class GoogleMapsGestures(private val googleMap: GoogleMap) : Gestures {
+
```

Comment:
```suggestion
val uiSettings get() = googleMap.uiSettings
```
I would create a getter for UI settings to not write `googleMap.uiSettings` every time. But it's up to you.